src/maybe_dangling.rs

```diff
@@ -1,11 +1,4 @@
-use {
-    ::core::{
-        mem::{ManuallyDrop as StdMD},
-    },
-    crate::{
-        ManuallyDrop,
-    },
-};
+use {crate::ManuallyDrop, ::core::mem::ManuallyDrop as StdMD};
```

Comment:
Since formatting totally kills the motivation behind the outermost `{}`:

```suggestion
use crate::ManuallyDrop;
use ::core::mem::ManuallyDrop as StdMD;
```